pkg/frontend/transport/handler.go

```diff
@@ -303,6 +350,38 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 	} else {
 		level.Info(util_log.WithContext(r.Context(), f.log)).Log(logMessage...)
 	}
+
+	var reason string
+	// 413, 422 or 429.
```

Comment:
nit: I don't think we need this comment :-) It's just redundant with the if statements.